packages/devtools_app/lib/src/shared/editor/editor_client.dart

```diff
@@ -59,6 +61,18 @@ class EditorClient extends DisposableController
           _supportsOpenDevToolsPage = isRegistered;
           _supportsOpenDevToolsForceExternal =
               capabilities?[Field.supportsForceExternal] == true;
+        } else if (method == LspMethod.editArgument.methodName) {
```

Comment:
nit: a switch statement may be easier to read than this long chain of if / elses

Reply:
It would be but unfortunately these are all getters, triggering the error: "Extension methods can't be used in constant expressions." (I'm guessing that is why this was written as a bunch of if statements in the first place)